src/index.js

```diff
@@ -1,18 +1,27 @@
 /* eslint-disable one-var */
 
 import intoStream from 'into-stream';
-import Geocoder from './geocoder';
-import GeocodeStream from './geocode-stream';
+import StandardGeocoder from './geocoder';
+import StandardGeocodeStream from './geocode-stream';
 
 /**
  * GeoBatch instance
  * @type {Function}
- * @param {Object} options The options for the GeoBatch
+ * @param {Object} options The options for the GeoBatch, default is
+ *                         {cacheFile: 'geocache.db',
+ *                           clientId: null,
+ *                           privateKey: null}
+ * @param {Object} Geocoder A geocoder.
+ * @param {Object} GeocodeStream A GeocodeStream.
  */
 export default class GeoBatch {
-  constructor({cacheFile, clientId, privateKey} =
-    {cacheFile: 'geocache.db', clientId: null, privateKey: null}) {
+  constructor(
+    {cacheFile = 'geocache.db', clientId = null, privateKey = null}
+      = {cacheFile: 'geocache.db', clientId: null, privateKey: null},
+    Geocoder = StandardGeocoder,
+    GeocodeStream = StandardGeocodeStream) {
```

Comment:
Break `) {` to a new line so it is easier to see the param block and the content block.